app/static-site/aws-s3/infra_config.go

```diff
@@ -0,0 +1,80 @@
+package aws_s3
+
+import (
+	"context"
+	"fmt"
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/cloudfront"
+	cftypes "github.com/aws/aws-sdk-go-v2/service/cloudfront/types"
+	nsaws "gopkg.in/nullstone-io/nullstone.v0/aws"
+	"gopkg.in/nullstone-io/nullstone.v0/contracts/aws-s3-site"
+	"log"
+	"os"
+	"strings"
+)
+
+// InfraConfig provides a minimal understanding of the infrastructure provisioned for a module type=aws-fargate
```

Comment:
this says fargate in the comment, I think you meant s3 (copy/paste error?)

Reply:
Good find